contracts/src/CrossChainApplications/NativeTokenBridge/tests/NativeTokenSourceTests.t.sol

```diff
@@ -216,16 +221,14 @@ contract NativeTokenSourceTest is Test {
         vm.expectRevert(_formatNativeTokenSourceErrorMessage("zero destination contract address"));
 
         new NativeTokenSource(
-            MOCK_TELEPORTER_MESSENGER_ADDRESS,
+            MOCK_TELEPORTER_REGISTRY_ADDRESS,
             _DEFAULT_OTHER_CHAIN_ID,
             address(0x0)
         );
     }
 
     function testInvalidTeleporterAddress() public {
-        vm.expectRevert(
-            _formatNativeTokenSourceErrorMessage("unauthorized TeleporterMessenger contract")
-        );
+        vm.expectRevert();
```

Comment:
Can we include the specific expected revert message here still?

Reply:
Fixed!